src/components/Accordion/AccordionIcon.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import cn from 'classnames';
+
+import classes from './AccordionIcon.module.css';
+import { useAccordionContext, AccordionIconVariant } from './Context';
+import { ReactComponent as Arrow } from './expand-collapse.svg';
+import { ReactComponent as CircleArrow } from './CircleArrow.svg';
+
+export interface AccordionIconProps {
+  height?: string;
+  width?: string;
+}
+
+export const AccordionIcon = ({
+  height = '20',
+  width = '20',
+}: AccordionIconProps) => {
```

Comment:
We're never changing the size of this icon, so this can be removed. I think its probably unlikely that it will be needed in the future as well. WDYT?

```suggestion
export const AccordionIcon = () => {
```